assets/app/view/game/player.rb

```diff
@@ -32,6 +32,10 @@ def render
           divs << h(Companies, owner: @player, game: @game, show_hidden: @show_hidden)
         end
 
+        if (minors = @game.player_card_minors(@player)).any?
```

Comment:
unless empty

Reply:
I keep forgetting to avoid any if possible. Will do.